idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -6,9 +6,9 @@ export default `
         <div class="details-header">
           <h1 data-hook="title"></h1>
           <div class="idea-controls">
-            <button data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
-            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span class="transition-text">Disabling...</span><span class="default-text">Disable <span data-hook="title"></span></span></button>
-            <button data-hook="install" id="install-button" class="button primary"><span class="state-change-inner"></span><span class="transition-text">Enabling...</span><span class="default-text">Enable <span data-hook="title"></span></span></button>
+            <button data-l10n-id="giveFeedback" data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
+            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span data-l10n-id="disableExperimentTransition" class="transition-text">Disabling...</span><span data-l10n-id="disableExperiment" class="default-text">Disable <span data-hook="title"></span></span></button>
```

Comment:
@lmorchard If we're intending to localize the individual experiment names, will this part (scrolled off the screen) work?

`<span data-l10n-id="disableExperiment" class="default-text">Disable <span data-hook="title"></span></span>`

Back when I did fxa localization on Gaia (l10n.js era), IIRC the flow was like this:
- Replace any nested html with a mustache inside the localized property: `disableExperiment=Disable {{ title }}`
- Manually call the l10n function to translate 'title' and 'Disable'
- Concat localized strings and html by building up a string
- Finally, insert into the DOM

One major reason for not embedding html in the properties files was to keep html invisible to localizers, since they might easily typo the tag when copying it over. Another is that putting html in properties files makes future html updates dependent on updating localizations.

I may be misremembering part of this flow, @stas should know more.

Unfortunately, the l20n.js docs don't mention nested html and localized text, so it's not clear what the recommended approach would be.